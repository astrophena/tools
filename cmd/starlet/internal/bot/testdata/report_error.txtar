update: forwarded
-- bot.star --
load("@starlet//tg.star", "tg")

def handle(update):
    # Invalid arguments, should cause and report an error.
    tg.send_message(config.owner_id)
