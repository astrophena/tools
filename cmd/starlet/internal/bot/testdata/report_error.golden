[
  {
    "method": "sendMessage",
    "args": {
      "chat_id": 123456789,
      "entities": [
        {
          "length": 163,
          "offset": 25,
          "type": "pre"
        }
      ],
      "link_preview_options": {
        "is_disabled": true
      },
      "text": "‚ùå Something went wrong:\n\nTraceback (most recent call last):\n  //bot.star:5:20: in handle\n  @starlet//tg.star:33:1: in _send_message\nError: function _send_message missing 1 argument (text)\n\n"
    }
  }
]